lib/widgets/bubbles.dart

```diff
@@ -5,85 +5,78 @@ import 'package:flutter/material.dart';
 import 'package:bars_frontend/main.dart';
 import 'package:bars_frontend/utils.dart';
 import 'package:bars_frontend/predictions.dart';
-import 'topBubbleBar.dart';
+import 'bubblesPrototype.dart';
 import 'dialogs.dart';
 
+/// Represents a draggable widget that includes a widget of type [FeatureBubble].
```

Comment:
Should we write why this is needed? Also, rename it? The FeatureBubble is, to the user, a draggable bubble.

Reply:
Yes, but it's renamed to FeatureBubble now. Or what do you mean?

Reply:
I am talking about the namings of FeatureBubble and DragBubble, and why we need both and not just make FeatureBubbles draggable.

Reply:
Done in ef37c85.